zenoh/__init__.py

```diff
@@ -12,3 +12,89 @@
 #   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
 #
 from .zenoh import *
+
+_INT8_MIN = -(1 << 7)
+_INT16_MIN = -(1 << 15)
+_INT32_MIN = -(1 << 31)
+_INT64_MIN = -(1 << 63)
+_INT128_MIN = -(1 << 127)
+
+_INT8_MAX = 1 << 7
+_INT16_MAX = 1 << 15
+_INT32_MAX = 1 << 31
+_INT64_MAX = 1 << 63
+_INT128_MAX = 1 << 127
+
+_UINT8_MAX = 1 << 8
```

Comment:
As the naming stands, these values are not correct. `_UINTX_MAX` should be actually `(1 << X) - 1`.

Reply:
Same for `_INTX_MAX` btw, but I use `<` for max, so it's ok, as they are private constants.